packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
Unfortunately this pull request doesn't implement with `.ios.jsx`. But, when/if it does, it raises more questions like what happens if you have both `ios.js` and `ios.jsx`?

Sorry to chime in super late but I actually think that this is harmful to support .jsx extension. We now live in a world where we have many transforms for es6, flow, jsx... Why would we add `.jsx` but not `.flow` or `.es6`. Supporting more extensions is only going to fragment tools even more.

Reply:
@vjeux your first comment makes sense if you wouldn't talk about React ecosystem. I am having a hard time trying to understand why would you use that argument.

`.jsx` was invented by React team from the beginning of the time where React appear on the market as long as I remember from 2014 (4 years ago).

So yeah, I wouldn't support any random extension that appear but this one do not makes sense at all, React Native is about React, so probably `.jsx` would be more important than `.js` itself, but because we started using Babel we forget where we come from.

But but, that is just an observation, I really don't care about this, outside that `airbnb` code style and IDE/Editors likes that extension.

Reply:
temp solution. but we need built-in one!
http://www.fallingcanbedeadly.com/posts/enabling-react-native-jsx-extension